fluent/test/functions_builtin_test.js

```diff
@@ -99,24 +99,19 @@ suite('Built-in functions', function() {
 
       errors = [];
       msg = bundle.getMessage('num-decimal');
-      assert.strictEqual(bundle.formatPattern(msg.value, args, errors), '{NUMBER()}');
-      assert.strictEqual(errors.length, 1);
-      assert.ok(errors[0] instanceof TypeError);
-      assert.strictEqual(errors[0].message, "Invalid argument type to NUMBER");
+      assert.strictEqual(bundle.formatPattern(msg.value, args, errors), '1,475,107,200,000');
+      assert.strictEqual(errors.length, 0);
 
       errors = [];
       msg = bundle.getMessage('num-percent');
-      assert.strictEqual(bundle.formatPattern(msg.value, args, errors), '{NUMBER()}');
-      assert.strictEqual(errors.length, 1);
-      assert.ok(errors[0] instanceof TypeError);
-      assert.strictEqual(errors[0].message, "Invalid argument type to NUMBER");
+      assert.strictEqual(bundle.formatPattern(msg.value, args, errors), '147,510,720,000,000%');
+      assert.strictEqual(errors.length, 0);
 
       errors = [];
       msg = bundle.getMessage('num-bad-opt');
-      assert.strictEqual(bundle.formatPattern(msg.value, args, errors), '{NUMBER()}');
+      assert.strictEqual(bundle.formatPattern(msg.value, args, errors), '1475107200000');
```

Comment:
This value comes from here: https://github.com/projectfluent/fluent.js/blob/d3022dec7a68020e0dc0a3d295c0261ec4b21e71/fluent/src/types.js#L97

The variable reference resolved fine, but the bad option caused `toString` to go into the error branch. The error is correctly reported, but I'm not sure if just `return this.value` is the right fallback. We need to return a string here, so `FluentNone` is not option.

Reply:
Perhaps simulating the result of a `FluentNone` would be better? I.e. `return "{???}"`.

Reply:
Or, we could format using some reasonable defaults.

Reply:
In 81cbe84 I changed this to return safe defaults.